xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/WebHomeSheet.xml

```diff
@@ -434,31 +427,44 @@
     <property>
       <code>#template('colorThemeInit.vm')
 .theme {
-  border: 1px dotted $theme.borderColor;
-  float: left;
-  margin: 0 1em 1em 0;
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  border: 1px solid $theme.borderColor;
   padding: 2px 4px;
   width: 300px;
+  /* Even when the content changes, takes all the space given to it in the grid of themes. */
+  height: 100%;
 }
 
 .current-theme {
   background-color: $theme.highlightColor;
 }
 
 .theme-info {
-  margin-left: 60px;
   color: $theme.textSecondaryColor;
   font-size: .9em;
 }
 
 .theme-info h3{
   margin-top: 0;
-  border-bottom: 1px solid $theme.borderColor;
 }
 
-/* Clear the floatting strategy after the display of all themes */
+/* Clear the floating strategy after the display of all themes */
 .theme-viewer h2, .theme-creation-form {
-  clear: both;
+  display: flex;
+  gap: .5rem;
+  align-items: center;
+  margin-bottom: 1em;
+}
+
+.theme-viewer ul {
+  /* We remove the default UL styles since we want to use a flex display to make a grid of items.*/
+  list-style: none;
```

Comment:
We wouldn't need to define this if the elements weren't list elements in the first place.

Reply:
I'd rather keep HTML semantic and apply the changes we need on CSS. It's only a couple rules :)
If for some reason the CSS cannot get loaded, the content would degrade way more nicely and the users would be able to tell what elements are part of a list.

Reply:
Thanks, explained by the other discussion.

Reply:
![image](https://github.com/user-attachments/assets/a0ed8c30-5e1c-4a0e-92c5-79ac7ee7778a)

I tested to see what it'd look like without CSS.
We do not have any rule about progressive enhancement  but I think it's still a small plus :)